src/main/java/sirius/web/data/ExcelExport.java

```diff
@@ -117,9 +236,32 @@ private void addCell(HSSFRow row, Object obj, int columnIndex, HSSFCellStyle sty
             cell.setCellValue(((BigDecimal) obj).doubleValue());
             return;
         }
+        if (obj instanceof ImageHeaderCol) {
+            cell.setCellValue(new HSSFRichTextString(((ImageHeaderCol) obj).name));
```

Comment:
this could also be done reading the first / last / every ImageCell - kinda redundant